website/docs/d/nsxt_ip_set.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_ip_set"
+sidebar_current: "docs-vcd-datasource-nsxt-ip-set"
+description: |-
+  Provides a datasource to read NSX-T IP Set. IP sets are groups of objects to which the firewall rules apply. 
+  Combining multiple objects into IP sets helps reduce the total number of firewall rules to be created.
+---
+
+# vcd\_nsxt\_ip\_set
+
+Supported in provider *v3.3+* and VCD 10.1+ with NSX-T backed VDCs.
+
+Provides a datasource to read NSX-T IP Set. IP sets are groups of objects to which the firewall rules apply. Combining
```

Comment:
I believe we have been using "data source" (two words) before?

```suggestion
Provides a data source to read NSX-T IP Set. IP sets are groups of objects to which the firewall rules apply. Combining
```

Reply:
True. I slipped here. Now I have changed here and in other places I could grep for docs.